### ISSUE TITLE
Why the losses for positive and negative samples are calculated in the same way?

### ISSUE THREAD

**Issue by goldfishxiao** [XLearning-SCU/2022-CVPR-AirNet#23]
`l_pos = torch.einsum('nc,nc->n', [q, k]).unsqueeze(-1)

l_neg = torch.einsum('nc,ck->nk', [q, self.queue.clone().detach()])

logits = torch.cat([l_pos, l_neg], dim=1)

logits /= self.T

labels = torch.zeros(logits.shape[0], dtype=torch.long).cuda()`

In MOCO.py, the similarity between the query sample and the positive and negative samples is l_pos and l_neg, respectively. Afterwards, logits and labels with all zeros are used to calculate the cross entropy loss. This is confusing because it is necessary to make l_pos greater and l_neg smaller.
